<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabInputs">
       <attribute name="title">
        <string>Inputs</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>703</width>
             <height>445</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QGroupBox" name="groupBoxRequired">
              <property name="title">
               <string>Requred</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="3" column="2">
                <widget class="QPushButton" name="pushButtonScenario">
                 <property name="maximumSize">
                  <size>
                   <width>120</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Change</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="labelRule">
                 <property name="text">
                  <string>Rule Name</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="QPushButton" name="pushButtonInputDirectory">
                 <property name="maximumSize">
                  <size>
                   <width>120</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Change</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="labelScenario">
                 <property name="text">
                  <string>Scenario Name</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="labelInputDir">
                 <property name="text">
                  <string>Directory Name</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QLabel" name="labelRuleName">
                 <property name="frameShape">
                  <enum>QFrame::Box</enum>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QLabel" name="labelScenarioName">
                 <property name="frameShape">
                  <enum>QFrame::Box</enum>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="2" column="2">
                <widget class="QPushButton" name="pushButtonRuleName">
                 <property name="maximumSize">
                  <size>
                   <width>120</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Choose</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLabel" name="labelInputDirName">
                 <property name="frameShape">
                  <enum>QFrame::Box</enum>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QPushButton" name="pushButtonInputFile">
                 <property name="text">
                  <string>Change</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLabel" name="labelInputFileName">
                 <property name="frameShape">
                  <enum>QFrame::Box</enum>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="labelInputFile">
                 <property name="text">
                  <string>File Name (.PRT file)</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBoxDailyValues">
              <property name="text">
               <string>Daily Values provided in files</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBoxOptional">
              <property name="title">
               <string>Optional</string>
              </property>
              <layout class="QGridLayout" name="gridLayout">
               <item row="4" column="1">
                <widget class="QComboBox" name="comboBoxInputs">
                 <item>
                  <property name="text">
                   <string>Flow only (NOSPILL)</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>All inputs (ALL)</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QPushButton" name="pushButtonRuleFileName">
                 <property name="maximumSize">
                  <size>
                   <width>120</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Change</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1" rowspan="2">
                <widget class="QLabel" name="labelDataDirName">
                 <property name="frameShape">
                  <enum>QFrame::Box</enum>
                 </property>
                 <property name="text">
                  <string>HydSim</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="QPushButton" name="pushButtonDataFileName">
                 <property name="maximumSize">
                  <size>
                   <width>120</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Change</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLabel" name="labelRuleFileName">
                 <property name="frameShape">
                  <enum>QFrame::Box</enum>
                 </property>
                 <property name="text">
                  <string>Rules.pm</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="labelDataFile">
                 <property name="text">
                  <string>Data File Name</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0" rowspan="2">
                <widget class="QLabel" name="labelDataDIr">
                 <property name="text">
                  <string>Data Directory</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="2">
                <widget class="QPushButton" name="pushButtonDataDirName">
                 <property name="maximumSize">
                  <size>
                   <width>120</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Change</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="labelInputs">
                 <property name="text">
                  <string>Inputs from file(s)</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="labelRuleFile">
                 <property name="text">
                  <string>Rule File Name</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLabel" name="labelDataFIleName">
                 <property name="frameShape">
                  <enum>QFrame::Box</enum>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacerInput">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabOutputs">
       <attribute name="title">
        <string>Outputs</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="2" column="0">
         <widget class="QLabel" name="labelDebug">
          <property name="text">
           <string>Debug outputs from COMPASS runs</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="labelOutputDir">
          <property name="text">
           <string>Ouput Directory</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="comboBoxTransport">
          <item>
           <property name="text">
            <string>None - do not generate</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>None - always on (CONST)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>MAF</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Flow</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Temperature </string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelTransport">
          <property name="text">
           <string>Generate fish Transport based on</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="pushButtonOutputDirectory">
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Change</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <spacer name="verticalSpacerOutput">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="comboBoxDebug">
          <item>
           <property name="text">
            <string>None</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Temperature</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Spill</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>All</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="labelOutputDirName">
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="text">
           <string>/pre-proc</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabRun">
       <attribute name="title">
        <string>Run</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="3" column="0" colspan="3">
         <layout class="QHBoxLayout" name="horizontalLayoutRun">
          <item>
           <spacer name="horizontalSpacerRun1">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonRun">
            <property name="text">
             <string>Run</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonCancelRun">
            <property name="text">
             <string>Cancel Run</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacerRun2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="pushButtonChangeTemp">
          <property name="text">
           <string>Change</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="3">
         <widget class="QFrame" name="frameConOut">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="lineWidth">
           <number>1</number>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QPlainTextEdit" name="plainTextEditOut">
             <property name="frameShape">
              <enum>QFrame::Box</enum>
             </property>
             <property name="lineWrapMode">
              <enum>QPlainTextEdit::NoWrap</enum>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelCompassb">
          <property name="text">
           <string>COMPASS executable </string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="pushButtonChangeCompassb">
          <property name="text">
           <string>Change</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelTemp">
          <property name="text">
           <string>Temperature model</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="labelTempExe">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="text">
           <string>temp_paulsen_r.exe</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="labelCompassbExe">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="text">
           <string>compassb.exe</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>747</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actioncompassb"/>
    <addaction name="actionAlternateTempModel"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuTools"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionHelp">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actioncompassb">
   <property name="text">
    <string>compassb</string>
   </property>
  </action>
  <action name="actionAlternateTempModel">
   <property name="text">
    <string>Alternate temp model</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>scrollArea</tabstop>
  <tabstop>pushButtonInputFile</tabstop>
  <tabstop>pushButtonInputDirectory</tabstop>
  <tabstop>pushButtonRuleName</tabstop>
  <tabstop>pushButtonScenario</tabstop>
  <tabstop>checkBoxDailyValues</tabstop>
  <tabstop>pushButtonRuleFileName</tabstop>
  <tabstop>pushButtonDataFileName</tabstop>
  <tabstop>pushButtonDataDirName</tabstop>
  <tabstop>comboBoxInputs</tabstop>
  <tabstop>pushButtonOutputDirectory</tabstop>
  <tabstop>comboBoxTransport</tabstop>
  <tabstop>comboBoxDebug</tabstop>
  <tabstop>pushButtonChangeCompassb</tabstop>
  <tabstop>pushButtonChangeTemp</tabstop>
  <tabstop>pushButtonRun</tabstop>
  <tabstop>pushButtonCancelRun</tabstop>
  <tabstop>plainTextEditOut</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
